single_cell/workflows/alignment_metrics/scripts/collect_metrics.py

```diff
@@ -181,8 +181,11 @@ def extract_insert_metrics(self):
         std_dev_ins_size = 0 if std_dev_ins_size == '?' else std_dev_ins_size
 
         return median_ins_size, mean_ins_size, std_dev_ins_size
-    
-    
+
+    def merge_biobloom_metrics(self):
+        merged = pd.merge(pd.read_csv(self.output), pd.read_csv("/biobloom_output/biobloom_count_metrics.csv"), left_index=True, right_index=True)
```

Comment:
this wont work when we parallelize over nodes as that file is not managed by pypeliner. 


make this file an output from align_pe... pass this as input to metrics... dont hardcode